dashboard/public/shared

```diff
@@ -0,0 +1 @@
+/Users/brent/git/cdo/shared/js/build/package
```

Comment:
actually, i think it should just not be committed (as it's added by rake install steps)